### ISSUE TITLE
Buzzing in Windows 11 with stereo output

### ISSUE THREAD

**Issue by LorenKeagle** [rebtd7/FF13Fix#91]
I've tried all of the other fixes mentioned regarding buzzing sounds.

Compatibility mode (any version) causes the game to immediately close after launch.  I'm unsure whether this is a crash or not.

I'm running a simple stereo output from my Realtek digital output.  No surround modes are available, and no spatial audio is enabled.  I've tried all of the different default audio formats available in sound settings, and none of them eliminate the buzz (although some of them do appear to reduce it slightly).

The buzzing seems to primarily be in the left speaker, but I haven't determined whether that's always the case (I've only tested in the opening menu).  

It may be unrelated, but when I first applied the fix, I did not have the buzzing issue.  I did have a different issue, where cutscene audio would play fine, but any in-game sequences would be completely silent.  I installed some seemingly unrelated updates, then after a reboot I started the game and heard the buzzing.

**Comment by LorenKeagle** [comment]
Aaaaand....

Now I'm back where I started.  I can only hear audio in the opening cinematic and cutscenes.  I get complete silence in the main menu, and in-game after loading a save.  

I can't get the buzzing to show up again, but I'm also not willing to play the game with no sound at all.  Not sure what to do next.  If I disable FF13Fix, or the 4Gb patch, the game no longer loads.  It either crashes/closes immediately, or sits and hangs until I kill the process. I've tried downgrading to earlier releases of FF13Fix, but the game stops working when I go too far back. (I did this to determine whether an older version of the mod would allow me to enable Compatibility Mode, but no dice)

I'm happy to provide any more info or run any test builds you care to provide.  Unfortunately, I only have stereo speakers available, so I can't test any 5.1 surround modes.

**Comment by LorenKeagle** [comment]
Some additional information:

If I enable 'Enhance Audio' on my audio device (same as "Enable audio enhancements" in the 'Advanced properties menu), I get the buzzing issue, but I can at least hear the game sound outside of cutscenes.

If I disable 'Enhance Audio' on my audio device, I do not hear buzzing, but all in-game audio is silent.

I've tried with every supported sound format to see if that makes a difference. 16 vs 24 bit formats do not seem to make a difference, but switching to a 192kHz sample rate does decrease the buzzing significantly.

**Comment by Yozora36** [comment]
Did you ever find a solution?

**Comment by LorenKeagle** [comment]
No.  Still running the interface at 192kHz and just turning the volume way down so that the buzz is not as noticeable.

**Comment by Nucleoprotein** [comment]
I have never noticed audio buzzing in FF13, can you record it or something?

**Comment by LorenKeagle** [comment]
Sorry, did not specify that this occurs in FFXIII-2.  I forgot that this package applies to the original game as well.

I've attached a desktop capture of the game launching to the game start screen to illustrate the buzzing in the left speaker.

https://user-images.githubusercontent.com/12399402/187964780-90eaffef-5f47-4638-b1f6-df96591fa2e7.mp4

**Comment by Nucleoprotein** [comment]
Somebody here: https://github.com/rebtd7/FF13Fix/discussions/95#discussioncomment-3530975 had similar problem and buzzing was caused by some Razer surround sound software.

If disabling Audio Enhancements in sound control panel disables all sound in system, then definitely you have issue with some audio software/drivers. That also can explain why it works in compatibility mode - in some modes Windows will emulate some sound factions and some audio effects are disabled.

I recommend uninstalling audio driver, then installing them clean. This should resolve this if this is not some terrible broken Nahimic Realtek extensions that are provided with some motherboards ...

**Comment by Nucleoprotein** [comment]
> 

Yeah, you have some software/driver installed with audio effects, i know Nahimic audio extensions cause problems in games.

**Comment by LorenKeagle** [comment]
That was some impressive inference! I couldn't find anything related to 'Nahimic' in my installed apps, but a registry search sure found something. Tracked it down to a pair of apps from 'A-Volute' named 'Sonic Studio 3' and 'Sonic Radar'.  They must have been installed via my motherboard vendor.  I removed these, disabled audio enhancements, rebooted, and I now have clean sound in-game.

To clarify something for others reading this thread, enabling 'Audio Enhancements' did not disable any system sounds. It only allowed FFXIII-2 to have no sound in-game.  Cut-scene and intro audio still worked fine, but the game was silent once I reached the main menu.  I discovered this combination of settings completely by accident, so don't think there was any real process besides throwing everything at the wall and seeing what stuck! :-)  Also, in my case, enabling any form of compatibility mode caused the game process to freeze at startup, so none of those workarounds were helpful here.
